pages/confirm-installation-popup.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, Locator, Page } from '@playwright/test';
```

Comment:
please remark that we import the type (seems to be something new added to Playwright documentation), see [discussion](https://github.com/jknphy/e2e-agama-playwright/pull/13/files#diff-cfee557db38eba21f19500b110f8c41734442f5ebdc71d1505c9f0cf296ad628R1)